Dockerfile

```diff
@@ -1,13 +1,13 @@
-FROM node:latest as build-stage
+FROM node:14.15.1 as build-stage
 WORKDIR /app
 COPY package*.json ./
 RUN npm install
 COPY ./ .
 RUN npm run build
 
-FROM nginx:1.17 as production-stage
+FROM nginx:1.18.0 as production-stage
 COPY nginx.conf /etc/nginx/nginx.conf
 RUN mkdir /app
 COPY --from=build-stage /app/dist /app
 EXPOSE 8080:8080
-CMD ["nginx", "-g", "daemon off;"]
\ No newline at end of file
+CMD ["nginx", "-g", "daemon off;"]
```

Comment:
Is this line needed in the other 2 UIs?

Reply:
I think so. It is nginx starting command.